plugins/completion/cmp/sources/default.nix

```diff
@@ -197,7 +197,7 @@ in
 {
   # For extra cmp plugins
   imports = [
-    ./codeium-nvim.nix
+    { cmpSourcePlugins.codeium = "codeium-nvim"; }
```

Comment:
This could be done here, but I think it makes more sense to do this within the `codeium-nvim.nix` module.